src/components/GroupPreview.js

```diff
@@ -4,24 +4,24 @@ import { Thread } from './Thread';
 
 export const GroupPreview = withRouter((props) => {
   let activeConversations
-  let threadTitles = []
+  const threadTitles = []
   if (props.threads.length > 0) {
     for (let i=0; i < props.threads.length && i < 3; i++) {
       threadTitles.push(
-        <li class="list-group-item">{props.threads[i].title}</li>
+        <li className="list-group-item">{props.threads[i].title}</li>
       )
     }
     activeConversations = <div>
-      <div class="card-header text-center">
+      <div className="card-header text-center">
         Active Conversations
       </div>
-      <ul class="list-group list-group-flush">
+      <ul className="list-group list-group-flush">
         {threadTitles}
       </ul>
     </div>
   }
 
-  let url = "/community/" + props.groupId
+  const url = `/community/${  props.groupId}`
```

Comment:
For this one, I would update the spaces in the string to something like this:
```suggestion
  const url = `/community/${props.groupId}`
```
Makes it a tad more readable that way.